SAMtags.tex

```diff
@@ -206,6 +206,8 @@ \subsection{Additional Template and Mapping data}
 \item[SA:Z:\tagregex{{\tt (}\emph{rname}{\tt ,}\emph{pos}{\tt ,}\emph{strand}{\tt ,}\emph{CIGAR}{\tt ,}\emph{mapQ}{\tt ,}\emph{NM}{\tt ;)}+}]
 Other canonical alignments in a chimeric alignment, formatted as a semicolon-delimited list.
 Each element in the list represents a part of the chimeric alignment. Conventionally, at a supplementary line, the first element points to the primary line.
+\emph{Rname} should be stated explicitly and is not permitted to be `{\tt=}'.
```

Comment:
should->must (I think this is what we said in the past about language...right?)

Reply:
I think "must" makes more sense in this sentence as "should" is simply a recommendation and is at odds with the "not permitted" bit later on (in RFC2119, although we don't follow this for the SAM spec).

However there was discussion at the meeting which made me wonder if it's valid anyway.  It is completely valid, and the only concerns are about remapping some but not all reads, but arguably that is an issue of the remapper not updating fields correctly.  If we want to make it so that is easy to do, then we should also be banning filling out TLEN, PNEXT, RNEXT, etc as they all have similar problems with remapping of partial data.

Comments?

Reply:
I think that we do not need to make reverting to the original data __easy__, we just need to make it __possible__ (without re-running the mapper). which is why I do not find it necessary to record the original TLEN, PNEXT, RNEXT as they are recoverable.

Reply:
Note this is SA tag and not OA tag.  The comment about = only applies if we edit the primary (or "previous") read.  In editing such data though we're already in a world of hurt when it comes to updating lots of fields, so I don't think = vs explicit is going to cut it.

I'll drop that sentence.